galaxy/values.yaml

```diff
@@ -86,6 +90,57 @@ jobHandlers:
     failureThreshold: 30
     timeoutSeconds: 5
 
+celery:
+  concurrency: 2
+  logLevel: "DEBUG"
```

Comment:
I recall Celery being chatty at `DEBUG`. Is this still the case? Should we bump this to `INFO`?